### ISSUE TITLE
Use accessibility labels only in debug mode

### ISSUE THREAD

**Issue by gayatrip** [kif-framework/KIF#459]
I want to set accessibility labels only in debug mode so that they do not go with the production release of project. I have tried using following code but its failing to set accessibility label to the element.
# ifdef DEBUG

```
 if(element && [element respondsToSelector:@selector(setAccessibilityLabel:)]) {
[element setAccessibilityLabel:label];
 }
```
# endif

Do you have any suggestions for addressing this case? Do you expect the KIF framework to be able to support this case?
